platform/darwin/src/MGLCircleStyleLayer.mm

```diff
@@ -10,6 +10,21 @@
 
 #include <mbgl/style/layers/circle_layer.hpp>
 
+namespace mbgl {
+
+    using namespace style;
```

Comment:
Nothing in this block is using the `mbgl::style` namespace, so this statement can go away.